nixos/tests/containers-extra_veth.nix

```diff
@@ -52,52 +52,59 @@ import ./make-test.nix ({ pkgs, ...} : {
 
   testScript =
     ''
-      $machine->waitForUnit("default.target");
-      $machine->succeed("nixos-container list") =~ /webserver/ or die;
+      machine.wait_for_unit("default.target")
+      out = machine.succeed("nixos-container list")
+      assert "webserver" in out
 
-      # Status of the webserver container.
-      $machine->succeed("nixos-container status webserver") =~ /up/ or die;
+      with subtest("Status of the webserver container"):
```

Comment:
in failure cases, the subtest title should ideally tell you what the high-level expectations are. While the container will have *some* kind of status, it is most helpful when the title says "Status of webserver container is UP".